File: 1) Arrays/easy/485. Max Consecutive Ones/Solution.java
class Solution {
  public int findMaxConsecutiveOnes(int[] nums) {
      int maxCons = 0;
      int currLen = 0;

      for (int i=0; i<nums.length; i++) {
          if (nums[i] == 1) {
              currLen += 1;
          } else {
              maxCons = Math.max(maxCons, currLen);
              currLen = 0;
          }
      }

      return Math.max(maxCons, currLen);
  }
}

File: 1) Arrays/easy/485. Max Consecutive Ones/Solution.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function(nums) {
  let max = 0;
  let currLen = 0;

  for (let i=0; i<nums.length; i++) {
      if (nums[i] == 1) {
          currLen += 1;
      } else {
          max = Math.max(max, currLen);
          currLen = 0;
      }
  }

  return Math.max(max, currLen);
};